src/pages/ProductsPage/Products.tsx

```diff
@@ -6,37 +6,107 @@ import ReactPaginate from "react-paginate";
 import Product from "../../components/ui/productCard/ProductCard";
 import Loader from "../../components/ui/loader/Loader";
 import { useFetchBreedsQuery } from "../../store/redux-paginstion-slice/fetch-slice";
-import { useAppDispatch } from "../../shared/utils";
+import { useAppDispatch, usePagination } from "../../shared/utils";
 import { changePage } from "../../store/redux-paginstion-slice/fetch-slice";
+import Icon from "../../components/ui/icons/icon";
+import { BreedType } from "../../types";
 
 export default function Products() {
+  const _ = require("lodash");
```

Comment:
wrong require

Reply:
you should import all the dependencies in the top of the file

also, import lodash.debounce and not the  whole lodash library